0.17/files/docker-entrypoint.sh

```diff
@@ -70,4 +85,13 @@ exec $SU_EXEC /opt/factorio/bin/x64/factorio \
   --server-adminlist "$CONFIG/server-adminlist.json" \
   --rcon-password "$(cat "$CONFIG/rconpw")" \
   --server-id /factorio/config/server-id.json \
-  "$@"
+)
+
+if [ "$ENABLE_SERVER_LOAD_LATEST" == "true" ]; then
+    FLAGS+=( --start-server-load-latest )
+else
+    FLAGS+=( --start-server "$SAVE_NAME" )
+fi
+
+# shellcheck disable=SC2086
+exec $SU_EXEC /opt/factorio/bin/x64/factorio "${FLAGS[@]}" "$@"
```

Comment:
I totally forgot that ${[@]} existed! Looks why better!

Reply:
Yep, and necessary now that some flags are conditionally added.